publish/dir.props

```diff
@@ -21,24 +21,24 @@
       <ConditionalErrorMessage>Pushing took too long</ConditionalErrorMessage>
     </IgnorableErrorMessages>
   </ItemGroup>
-  
+
   <ItemGroup>
-    <CompressedFile Include="$(PackagesOutDir)**/*$(CompressedFileExtension)">
+    <CompressedFile Include="$(PackageOutputPath)**/*$(CompressedFileExtension)">
```

Comment:
Should be `AssetOutputPath`.  In general - .nupkgs go to `PackageOutputPath`.  Everything else goes to `AssetOutputPath`.